task3/src/routes/autoSuggest/index.js

```diff
@@ -1,17 +1,28 @@
 import express from 'express';
 
 import validators from './validators';
-import { getFilteredSliceOfUsers } from '../../usersData';
+import UserService from '../../services/userService';
+import { UserModel } from '../../models';
 
+const userService = new UserService(UserModel);
 const router = express.Router();
 
 router.get(
     '/getAutoSuggestUsers',
     validators.getAutoSuggestUsersQuery,
-    (req, res) => {
-        const { limit, loginSubstring } = req.query;
-        const data = getFilteredSliceOfUsers(limit, loginSubstring);
-        res.json(data);
+    async (req, res) => {
+        try {
+            const { limit, loginSubstring } = req.query;
+            const data = await userService.getFilteredSliceOfUsers(
+                limit,
+                loginSubstring
+            );
+            res.json(data);
+        } catch (err) {
+            res.status(err.status || 500).json({
```

Comment:
Usually errors are passed down to some error handling middleware

Reply:
Will try to fix in task4